robopi:
  actuator_interface:
    loop_hz: 10 # hz
    interface_type: "velocity"
    wheels:
      - wheel_left
      - wheel_right
  gpio_wheels:
    #from gpio wiring
    wheel_left:
      en: 21
      inF: 20
      inB: 16
    wheel_right:
      en: 13
      inF: 19
      inB: 26